jetty-core/jetty-compression/jetty-compression-server/src/main/java/org/eclipse/jetty/compression/server/CompressionConfig.java

```diff
@@ -178,7 +178,7 @@ public Set<String> getCompressIncludeMimeTypes()
      * @see #getCompressExcludePaths()
      */
     @ManagedAttribute("Set of Response Compression Path Exclusions")
-    public Set<String> getCompressPathIncludes()
+    public Set<String> getCompressIncludesPath()
```

Comment:
Rename to `getCompressIncludesPaths()`, plural like the others.

Reply:
Doh! bad plural. :-/